Packs/MailListener/Integrations/MailListenerV2/MailListenerV2_test.py

```diff
@@ -109,8 +110,61 @@ def test_convert_to_incident():
                 '4:*'
             ]
         ),
+        (
+            datetime(year=2020, month=10, day=1),
+            'SomeIMAPServer',
```

Comment:
Why do we need this example? When will we get "SomeIMAPServer"?

Reply:
It is unnecessary.